packages/driver/cypress/e2e/cypress/stack_utils.cy.js

```diff
@@ -93,6 +126,15 @@ describe('driver/src/cypress/stack_utils', () => {
 
       expect($stackUtils.getCodeFrame(originalErr)).to.be.undefined
     })
+
+    it('returns relativePath from root when `absoluteFile` starts with `repoRoot`', () => {
+      // question - how do i mock a value in the cypress config?
+      cy.stub($sourceMapUtils, 'getSourceContents').returns(sourceCode)
+      cy.spy($stackUtils, 'getCodeFrameFromSource')
+      $stackUtils.getCodeFrame(originalErr)
+      // question - i know this is being called b/c my console log is showing up
+      expect($stackUtils.getCodeFrameFromSource).to.have.been.called
```

Comment:
Here I spy on `getCodeFrameFromSource`. I know it's being called during the test because if I throw a console log inside `getCodeFrameFromSource`, it shows up. However, my assertion on line 136 fails 🥲 ? por que?

Reply:
Could be a red herring, but I think the problem here is related to our compilation process (webpack, babel, typescript etc...)

I put a debugger. We hit this point twice; once during the original call in your test and once during the actual failed assertion. It looks like the first time it points to the correct stub, but the second time it's slightly different, you can see it's got `__WEBPACK__`...

![image](https://user-images.githubusercontent.com/19196536/190592588-bb9fdb78-646d-4d67-a1d6-02ec0a8424cb.png)

![image](https://user-images.githubusercontent.com/19196536/190592702-3c5c4967-f0c4-4c85-b438-077ff52b68fd.png)

Reply:
I think what's happening is you are stubbing the function, but when the code actually runs it's calling the webpack module cache version, which is possibly immutable (or just the cached version w/o the `cy.spy` wrapper).

Reply:
Either way, I think we don't even need this - how about doing this? https://github.com/cypress-io/cypress/compare/update-git-info...lachlan/update-git-info?expand=1